attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
We can take it out, but then it'll just fall back to this: https://github.com/DataDog/dd-agent-builder/blob/master/attributes/default.rb#L3